bin/setup-database.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
I recommend starting the script with:
```
#!/bin/bash
set -euo pipefail
```

This gives us some nice bash features for future use and ensures the script won't continue executing if an error occurs.